app/models/ssh_execution_provider.rb

```diff
@@ -55,8 +59,12 @@ def cockpit_url_for_host(host)
 
     private
 
-    def ssh_user(host)
-      host.host_param('remote_execution_ssh_user')
+    def ssh_user(template_invocation, host)
```

Comment:
This would break cockpit integration, `ssh_port` is being called from https://github.com/theforeman/foreman_remote_execution/pull/703/files#diff-bac7fac9f5249d58891a94052593b2938f354d1f71469b149d3bf1b774fa333bR49 where no template invocation is available. Maybe reorder the arguments and make template_invocation optional?